dockerfiles/eclccserver/Dockerfile

```diff
@@ -20,6 +20,9 @@
 ARG BUILD_LABEL
 FROM hpccsystems/platform-core:${BUILD_LABEL}
 
+USER root
```

Comment:
Possibly should be in a base image

Reply:
Also may need to make sure RBAC is properly set up to allow eclccserver to do this - it's working for me but I may have changed the default access controls. I'll raise a Jira to check that.

Reply:
Also may need to make sure RBAC is properly set up to allow eclccserver to do this - it's working for me but I may have changed the default access controls. I'll raise a Jira to check that.